README.md

```diff
@@ -36,24 +36,18 @@ In the demo client [repository](https://github.com/tecton-ai/TectonClientDemo) u
 jar that you generate from this repo using `./gradlew clean build`.
 
 1. Change the dependencies target to this and point the files attribute to your java client jar.
-2. Add all dependencies from `tecton-http-client-java` except `testImplementation` to `build.gradle` in the demo client repository.
+2. Add the okhttp3 dependency from `tecton-http-client-java` to `build.gradle` in the demo client repository. This is
+   necessary because it is an API dependency, but JAR files don't include information about transitive dependencies.
+   Published libraries on Maven don't have this issue.
 
 ```
 dependencies {
     implementation files('libs/java-client-0.1.0-SNAPSHOT.jar')
-    implementation 'com.google.code.gson:gson:2.2.4'
-    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
-    testImplementation 'junit:junit:4.13.2'
     // OkHttp Client
     implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.10.0'
-    // Moshi JSON Parsing Library
-    implementation group: 'com.squareup.moshi', name: 'moshi', version: '1.13.0'
-    // Moshi JSON Adapter Library for Standard Java Types
-    implementation group: 'com.squareup.moshi', name: 'moshi-adapters', version: '1.13.0'
```

Comment:
is moshi not needed anymore ? i haven't tried on your branch but i added these instructions to the readme after failing previously: https://github.com/tecton-ai/tecton-http-client-java/pull/82

Reply:
Left them, with a comment.